plugins/graphql-java-codegen-gradle-plugin/graphql-codegen-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id "com.gradle.plugin-publish" version "0.11.0"
+    id "java-gradle-plugin"
+}
+
+repositories {
+    maven {
+        url 'https://repo.gradle.org/gradle/libs-releases-local'
+    }
+    mavenCentral()
+    mavenLocal()
+    jcenter()
+}
+
+apply plugin: "java"
+apply plugin: "idea"
+apply plugin: "maven-publish"
+
+group = "io.github.kobylynskyi"
+version = "1.4.2"
+description = "Provides a task for generating Java code based on GraphQL schema"
+
+dependencies {
+    compile gradleApi()
+
+    compile 'io.github.kobylynskyi:graphql-java-codegen:1.4.2'
```

Comment:
This still means that the library's artifact needs to be published (unless I missed something).
We could use an actual multi-project gradle build (and declare dependencies like `compile project(":graphql-java-codegen")`.
However, I'm not sure about the maven one. I don't know how to setup a Gradle build for the maven plugin subproject

Reply:
You are right.
After some research, I was not able to find a good solution for importing a Maven project inside a Gradle project or building a maven plugin inside Gradle. We can leave this as an open question and make an improvement later.

But for right now I've added a [Development process](https://github.com/kobylynskyi/graphql-java-codegen/blob/89df7a3332e99011d7f953ffc4d0fc495193e586/CONTRIBUTING.md#development-process) instructions which explain how to do the changes the plugins without publishing the core library to maven central.
@joffrey-bion Please let me know what you think. Thanks!